rclcpp/test/rclcpp/node_interfaces/test_node_graph.cpp

```diff
@@ -66,18 +67,57 @@ class TestNodeGraph : public ::testing::Test
 
   const rclcpp::node_interfaces::NodeGraph * node_graph() const {return node_graph_;}
 
+  size_t get_num_topics()
+  {
+    constexpr std::chrono::milliseconds timeout(100);
+
+    int tries = 0;
```

Comment:
nitpick: `size_t`?

Reply:
Yeah, I guess that is more correct.  I'll switch it.